index.html

```diff
@@ -4,25 +4,63 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Manage books using objects</title>
+    <title>Manage Books</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We"
+      crossorigin="anonymous"
+    />
     <script src="./js/main.js" defer></script>
   </head>
   <body>
```

Comment:
- [ ] Your design interface does not match  the provided [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m3_style_core_elements.md). Comparing your design with the wireframe template, can can observe that:
- The heading title `All awesome books` appears at the center of the page
- The `remove buttons` appear at the extreme right of each table row
- A form comes with the heading title `Add a new book`
- A narrower `form inputs`
- An `add button` to the right of the form

 We actually want our designs to mimic the provided wireframe as much as possible. Kindly ensure to improve on the highlighted points in an attempt to satisfy a client's demands.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/129119743-c5d0325c-0a01-48ae-a5f6-b00613c0cee2.png)


**The original wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/129119852-bfd48aee-25dd-44fa-9ccb-024f86c4ae6d.png)